edge-agent/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,23 +1,22 @@
 # syntax=docker/dockerfile:1.4
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0-focal-arm32v7
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-alpine
 
-RUN apt-get update && \
-    apt-get install -y libsnappy1v5 && \
-    ln -s /lib/arm-linux-gnueabihf/libdl.so.2 /usr/lib/arm-linux-gnueabihf/libdl.so && \
-    ln -s /lib/arm-linux-gnueabihf/libc.so.6 /usr/lib/arm-linux-gnueabihf/libc.so && \
-    rm -rf /var/lib/apt/lists/*
+ARG EXE_DIR=.
+
+# RocksDB requires snappy
+RUN apk update && \
+    apk add --no-cache snappy
 
-# create a user to allow agent to optionally run as non-root
+# create a default user to allow agent to run as non-root
 # value was chosen as a large value to avoid a typical regular uid, 
 # and matches the "edgeagentuser" created in install packages.
 ARG EDGEAGENTUSER_ID
 ENV EDGEAGENTUSER_ID ${EDGEAGENTUSER_ID:-13622}
-RUN useradd -ms /bin/bash -u ${EDGEAGENTUSER_ID} edgeagentuser 
+RUN adduser -Ds /bin/sh -u ${EDGEAGENTUSER_ID} edgeagentuser 
```

Comment:
What does this change do?

Reply:
The old one is the appropriate call on an Ubuntu system, the new one is the appropriate (and equivalent) call on an Alpine system.